### DISCUSSION TITLE
How to find out version thinq you have?

### DISCUSSION THREAD

**Discussion by monsieurlatte** [nVuln/homebridge-lg-thinq#187] [Q&A]
I have a LDP6797BD dishwasher and I'm not able to figure out the thinq version to see if it's supported?

**Comment by nVuln** [comment]
turn on homebridge debug mode and you can see device data in debug logging (data included thinq version)
<img width="453" alt="homebridge debug mode" src="https://user-images.githubusercontent.com/54855446/193384238-33e3fbc2-495b-4a61-96be-da8ca11b60e9.png">

I still working to implement as much as I can, even thinq v1
if your dishwasher using thinq v1, please open new issue with device data above, I'll try my best to implement it when I have time
